src/azure-cli/azure/cli/command_modules/serviceconnector/_validators.py

```diff
@@ -384,6 +385,19 @@ def get_missing_source_args(cmd):
 
     return missing_args
 
+def get_missing_source_create_args(cmd, namespace):
```

Comment:
just a note, may be not the scope of this PR. 
the logic for interactive experience is getting more and more confusing now, need refactoring.